tests/create_test_dataset.py

```diff
@@ -149,11 +167,7 @@ def progressBar(i, tot):
 print("Filename: {0}".format(filename))
 
 if download_data==True: 
-    try: 
-        test_dataset.to_netcdf(path=filepath, mode='w') # Output 
-        print("File saved to: {0}".format(filepath))
-    except:
-        print("You already have a file with the same name: "+filename+"\nDelete that file to output a new one with the same name.")
```

Comment:
I did kind of like having a descriptive print statement shown if the download failed due to a pre-existing file with the same name. Am open to removing it like you did here, but curious why you didn't like it?

Reply:
mm this wasn't working for me properly. I think the file paths weren't the same for me for some reason. Also it was catching all errors (not just file-already-exists errors)... but I don't have strong thoughts, I can add it back and we can revisit later!